app/models/person.rb

```diff
@@ -61,6 +61,15 @@ class Person < ActiveRecord::Base
   STATUS = ['Leave of Absence', 'Inactive', 'Active', 'Applicant','Prospect','Declined']
   DEPARTMENT = ['Police', 'CERT', 'Other']
 
+  after_initialize do
+    if self.new_record?
+      # values will be available for new record forms.
+      if self.department.nil? && Department.managing_people.count == 1
+        self.department = Department.managing_people.first
```

Comment:
This will need `Department.active.managing_people.first`, otherwise we may get an inactive department.

Reply:
Done